packages/dev/docs/pages/releases/2022-08-26.mdx

```diff
@@ -0,0 +1,244 @@
+
+Thanks to all our contributors for helping with this release!
+
+
+## New Components
+  ListView
+    - `@react-spectrum/list` - [Docs](https://react-spectrum.adobe.com/react-spectrum/ListView.html)
+    - `@react-aria/gridlist` - [Docs](https://react-spectrum.adobe.com/react-aria/useGridList.html)
+
+## Fixes
+* Use id when provided in `useTable` and `useGrid` - [@moelmaghraby](https://github.com/moelmaghraby) - [PR](https://github.com/adobe/react-spectrum/pull/3367)
+* FocusScope to restore to first focusable element when no tabbable element in scope - [@majornista](https://github.com/majornista) - [PR](https://github.com/adobe/react-spectrum/pull/2763)
+* Escape key should continue propagation to close `Overlay` in `useComboBox` - [@majornista](https://github.com/majornista) - [PR](https://github.com/adobe/react-spectrum/pull/3307)
+* Stop propagation when pressing Enter on a native link - [@devongovett](https://github.com/devongovett) - [PR](https://github.com/adobe/react-spectrum/pull/3380)
+* Add AriaLabelingProps to `ComboBox`, `Calendar`, `RangeCalendar`, `Tabs`, and `StatusLight` - [@ktabors](https://github.com/ktabors) - [PR](https://github.com/adobe/react-spectrum/pull/3378)
+* Add `FocusRing` to `ListView` - [@ktabors](https://github.com/ktabors) - [PR](https://github.com/adobe/react-spectrum/pull/3168)
+* Update `FocusScope` to restore correctly for overlays - [@snowystinger](https://github.com/snowystinger) - [PR](https://github.com/adobe/react-spectrum/pull/3323)
+* Fix focus style on invalid inputs - [@reidbarber](https://github.com/reidbarber) - [PR](https://github.com/adobe/react-spectrum/pull/3384)
+* Refactor virtualizer persisted keys - [@devongovett](https://github.com/devongovett) - [PR](https://github.com/adobe/react-spectrum/pull/3394)
+* Remove extraneous getKeyLeftOf/getKeyRightOf from `ListLayout` - [@LFDanLu](https://github.com/LFDanLu) - [PR](https://github.com/adobe/react-spectrum/pull/3415)
+* Add className to DOMAttributes interface - [@reidbarber](https://github.com/reidbarber) - [PR](https://github.com/adobe/react-spectrum/pull/3409)
+* Prevent resize error when `DatePicker` input ref is undefined - [@LFDanLu](https://github.com/LFDanLu) - [PR](https://github.com/adobe/react-spectrum/pull/3413)
+* Make default `TextArea` grow based on text content unless there is an explicit height - [@devongovett](https://github.com/devongovett) - [PR](https://github.com/adobe/react-spectrum/pull/3379)
+* removing getRowText prop from useGridList - [@LFDanLu](https://github.com/LFDanLu) - [PR](https://github.com/adobe/react-spectrum/pull/3431)
+* Localize TimeField label in DatePicker and DateRangePicker - [@devongovett](https://github.com/devongovett) - [PR](https://github.com/adobe/react-spectrum/pull/3433) * REMOVE IF THIS DOESN"T COME IN
+* Remove millisecond granularity from date/time pickers - [@devongovett](https://github.com/devongovett) - [PR](https://github.com/adobe/react-spectrum/pull/3435)
```

Comment:
we didn't actually remove anything because it never worked... don't wanna scare people

```suggestion
* Remove millisecond granularity from types for date/time pickers - [@devongovett](https://github.com/devongovett) - [PR](https://github.com/adobe/react-spectrum/pull/3435)
```